src/Kunstmaan/GeneratorBundle/Generator/LayoutGenerator.php

```diff
@@ -30,70 +30,85 @@ class LayoutGenerator extends KunstmaanGenerator
     private $demosite;
 
     /**
+     * @var string
+     */
+    private $browserSyncUrl;
+
+    /**
      * Generate the basic layout.
      *
      * @param BundleInterface $bundle  The bundle
      * @param string          $rootDir The root directory of the application
      */
-    public function generate(BundleInterface $bundle, $rootDir, $demosite)
+    public function generate(BundleInterface $bundle, $rootDir, $demosite, $browserSyncUrl)
     {
         $this->bundle   = $bundle;
         $this->rootDir  = $rootDir;
         $this->demosite = $demosite;
+        $this->browserSyncUrl = $browserSyncUrl;
 
         $this->shortBundleName = '@'.str_replace('Bundle', '', $bundle->getName());
 
-        if ($this->demosite) {
-            $this->generateBowerFiles();
-        }
-        $this->generateGulpFiles();
+        $this->generateGroundcontrolFiles();
         $this->generateGemsFile();
         $this->generateAssets();
         $this->generateTemplate();
     }
 
     /**
-     * Generate the bower configuration files.
+     * Generate the groundcontrol(gulp) configuration files.
      */
-    private function generateBowerFiles()
+    private function generateGroundcontrolFiles()
     {
         $this->renderFiles(
-            $this->skeletonDir . '/bower/',
+            $this->skeletonDir . '/groundcontrol/bin/',
+            $this->rootDir . '/bin/frontend/',
```

Comment:
This was a proposal from Ben and Stijn, but I don't know if this is the ideal location for de groundcontrol files..
@Devolicious @diskwriter  Any thoughts?